pkg/api/resolver_model_tag_edit.go

```diff
@@ -0,0 +1,20 @@
+package api
+
+import (
+	"context"
+
+	"github.com/gofrs/uuid"
+	"github.com/stashapp/stash-box/pkg/models"
+)
+
+type tagEditResolver struct{ *Resolver }
+
+func (r *tagEditResolver) Category(ctx context.Context, obj *models.TagEdit) (*models.TagCategory, error) {
+	if obj.CategoryID == nil {
+		return nil, nil
+	}
+
+	qb := r.getRepoFactory(ctx).TagCategory()
+	categoryID, _ := uuid.FromString(*obj.CategoryID)
+	return qb.Find(categoryID)
```

Comment:
I recommend using a dataloader for this seeing as a scene could have a lot of tags, most of which will have the same categories. The tagResolver already uses one for categories.

Reply:
Duh, no, this is a tagEdit, nevermind.